src/transformers/utils/doc.py

```diff
@@ -1117,6 +1117,8 @@ def docstring_decorator(fn):
             code_sample = sample_docstrings["BaseModel"]
         elif "ImageClassification" in model_class:
             code_sample = sample_docstrings["ImageClassification"]
+        elif "VisionClassification" in model_class:
+            code_sample = sample_docstrings["ImageClassification"]
```

Comment:
Not sure why this is required.

Reply:
Test for docstring failed they use to search image classification in model to create docstring I guess which is why added it there sort of hack but need clarity. If not the docstring breaks